src/main/java/uk/gov/justice/digital/delius/data/api/InstitutionalReport.java

```diff
@@ -1,13 +1,20 @@
 package uk.gov.justice.digital.delius.data.api;
 
+import io.swagger.annotations.ApiModelProperty;
 import lombok.Builder;
 import lombok.Data;
 
+import java.util.Optional;
+
 @Data
 @Builder(toBuilder = true)
 public class InstitutionalReport {
     private Long institutionalReportId;
     private Long offenderId;
     private Conviction conviction;
-    private Sentence sentence;
+
+    @ApiModelProperty(notes = "Deprecated - Use conviction to access sentence")
```

Comment:
I will remove this property once New Tech is updated

Reply:
ok